privacyidea/static/templates/baseline.html

```diff
@@ -25,5 +25,11 @@
                 target="documentation">
             <translate>Online Documentation</translate>
         </a>
+       <a class="navbar-text pull-right pull-right"
+           ng-show="piGDPRLink"
+           href="{{ piGDPRLink }}"
+           target="_external">
+            <translate>Privacy Statement</translate>
```

Comment:
So if someone wants to change the link text they must change the baseline template?
And what about an imprint link? Should we provide a functionality for that as well?

Reply:
I think we should keep it simple. If people want to change more, they should change the baseline.html themselves.